gulpfile.js

```diff
@@ -151,6 +154,13 @@ gulp.task('exportPatterns', ['patternsExport:clean'], () => {
 
 });
 
+gulp.task('distributeSharedConfig', ['css:clean'], (cb) => {
+  exec('distribute shared config', () => {
+    distributeSharedConfig.distribute();
+  });
+  cb();
```

Comment:
Is it possible that this callback will be called before the file is written to disk?

Reply:
You could pass `cb` down through distribute, and have it be called at the `fs.writeFile` step, or promises/async/await. I guess it depends on if its being used shortly afterwards in the sass build/watch steps

Reply:
Good spot. `distribute-shared-config` will need more Promises to make this work, and I think that's the way to go.